rotkehlchen/tasks/manager.py

```diff
@@ -397,25 +401,38 @@ def _maybe_decode_evm_transactions(self) -> None:
     def _maybe_check_premium_status(self) -> None:
         """
         Validates the premium status of the account and if the credentials are not valid
-        it deactivates the user's premium status.
+        it retries for 3 times before deactivating the user's premium status, but if the
+        credentials are valid it will reactivate the user's premium status.
```

Comment:
```suggestion
        it retries 3 times before deactivating the user's premium status. If the
        credentials are valid and the premium status is not correct it will reactivate the user's premium status.
```